maps/kondaru.dmm

```diff
@@ -52211,15 +52211,11 @@
 /obj/disposalpipe/junction/left/south,
 /turf/simulated/floor/plating,
 /area/station/maintenance/south)
-"eFl" = (
-/turf/simulated/wall/auto/reinforced/supernorn/yellow,
-/area/station/mining/magnet)
+/obj/stool/bar,
```

Comment:
This is invalid syntax, no tile identifier precedes it.

Reply:
i think I messed up solving a merge conflict. I'll try and fix this

Reply:
should be fixed